src/style.css

```diff
@@ -1,3 +1,87 @@
+@import 'https://fonts.googleapis.com/css?family=Roboto:100,100italic,300,300italic,regular,italic,500,500italic,700,700italic,900,900italic';
+
 body {
   margin: 0;
 }
+
+.header{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.header__nav{
+  margin: 0 50px;
+}
+
+.header__logo{
+  margin: 10px 50px;
+  width: 40px;
+  height: 40px;
+}
+
+.header__nav-items{
+  padding: 0;
+  list-style: none;
+  display: flex;
+  margin: 0;
+  gap: 20px;
+}
+
+.header__nav-item{
+  text-decoration: none;
+  color: #000;
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
```

Comment:
move to `*` selector

Reply:
when I placed this selectors to "*", my title became a text part of site

Reply:
Please write it in fe_chat, because it shouldn`t work like that